aasaan_jobs/aasaanjobs.py

```diff
@@ -1,20 +1,42 @@
 import logging
-import os
 import re
+import os
 import math
 import bs4
-from time import strptime
 from bs4 import BeautifulSoup
+import pandas as pd
+
 from kirmi import Kirmi
-# import pandas as pd
 
 website_baseurl = 'https://www.aasaanjobs.com'
 
-# logging.basicConfig(level=logging.DEBUG)
-# logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
 
 scraper = Kirmi(caching=True, cache_path="./aasan_cache.sqlite3")
-df_lst = []
+
+
+def save_to_csv(job_details_list, filename, page_num):
+
+    """
+    :param job_details_list: lst List of Job details dict
+    :param filename: str
+    :param page_num: int
+    :return:
+    """
+
+    logger.debug("Saving Job details list for {} - page {}".format(filename, str(page_num)))
+
+    df = pd.DataFrame(job_details_list)
+
+    filename = filename["category"].replace('/', "") + '.csv'
+
+    # if file does not exist write header
+    if not os.path.isfile(filename):
+        df.to_csv(filename, header='column_names', index=False, sep="|")
+    else:  # else it exists so append without writing the header
+        df.to_csv(filename, mode='a', header=False, index=False, sep="|")
+
```

Comment:
@yashodhan19 Thanks for working on this. A quick question. Do we need to have pipe separated values? `df.to_csv` with default values for `sep` argument would create a csv file. Are there fields with commas?

Reply:
Thanks, Raghu Some fields like the job description do have commas.

Reply:
Thanks for the clarification